architecture/PROXY_ROUTING.md

```diff
@@ -0,0 +1,147 @@
+# Proxy Routing
+
+The proxy chooses where to route a request primarily by the incoming Host. The routing is configured
```

Comment:
suggest more context like `by the incoming Host URL the client originally made their request to`